openstack/placement/values.yaml

```diff
@@ -47,11 +48,14 @@ uwsgi:
 max_pool_size: 10
 max_overflow: 5
 
-api_db: # Only used when mariadb.enabled=False to connect to the nova-api-mariadb
+api_db:  # Only used when mariadb.enabled=False to connect to the nova-api-mariadb
   name: placement
   user: placement
   password: null
 
+dbType: ""
+dbName: "placement"
```

Comment:
Do we need this if we set the `databases` and the `name` in the individual DB-specific values dicts?

Reply:
`dbName` isn't needed, `dbType` is, because `utils.db_url` and `proxysql` configuration logic depends on it

Reply:
`dbType` defaults to `mariadb` anyways, right?

When looking that the values as service-owner, it's confusing to have the db name in 3 places. Hence me trying to reduce "clutter" :) If you say it's necessary, we can keep it.